pam/go-loader/libpam-authd.pam-auth-update.in

```diff
@@ -0,0 +1,16 @@
+Name: Authd authentication
+Default: yes
+Priority: 128
```

Comment:
Are you sure priority 128 is enough? We need to be before pam_unix to bypass the password request.

Reply:
Oh, right... I think it has be increased, I need to check better this, I just used this as a placeholder for now.

Also I think we should be lower than smart-card access if that's enabled, so I need to check a bit the numbers around.